### DISCUSSION TITLE
Hosting costs for a large instance?

### DISCUSSION THREAD

**Discussion by jabba-jedi** [requarks/wiki#6806] [Help / Questions]
I'm thinking of self-hosting a large wiki instance (700 users, >8000 pages). Is there some kind of guidance/calculator to estimate how much with it cost me?

**Comment by NGPixel** [comment]
That depends on the hosting provider, the VM sizes, bandwidth, storage, etc. Not something that can be answered...

**Comment by jabba-jedi** [comment]
I hope someone hosting a wiki posts their rough spendings + the details of their setup, so please don't close the issue for a while :)
